### PR TITLE
CultureInfoCache.GetCultureInfo Should not be case sensitive

### PR DESCRIPTION
**Pull request by hishamco**
I made this PR to fix issue #95 
/cc @kirthik

### CODE REVIEW

**Comment by hishamco** [comment]
:watch:

File: src/Microsoft.Framework.Globalization.CultureInfoCache/CultureInfoList.cs [hunk 1]
```diff
@@ -19,34 +20,58 @@ public static partial class CultureInfoCache
         /// As new versions of .NET Framework and Windows are released, this list should be regenerated to ensure it
         /// contains the latest culture names.
         /// </summary>
-        public static readonly HashSet<string> KnownCultureNames = new HashSet<string>
+        public static readonly HashSet<string> KnownCultureNames = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
         {
             "",
+            "aa",
```
Comment (Eilon) [inline src/Microsoft.Framework.Globalization.CultureInfoCache/CultureInfoList.cs]: Any particular reason that ~400 extra cultures got added?

Comment (hishamco) [inline src/Microsoft.Framework.Globalization.CultureInfoCache/CultureInfoList.cs]: These cultures have been added after i ran ```CultureInfoGenerator``` command, i will remove them now

**Comment by hishamco** [comment]
The PR has been updated and rebased

**Pull request merged**